client/signup/config/flows.js

```diff
@@ -1,5 +1,6 @@
 import { isEnabled } from '@automattic/calypso-config';
 import { englishLocales } from '@automattic/i18n-utils';
+import { addQueryArgs as addWPQueryArgs } from '@wordpress/url';
```

Comment:
Going to refactor this to use Calypso's lib for consistency so we can remove this.